### PR TITLE
add actual button to application page for downloading to csv

### PR DESCRIPTION
**Pull request by jinpark**
For https://github.com/DjangoGirls/djangogirls/issues/101

Not sure if we compile `.styl` files when we deploy but I added the `css` file anyway. Not sure why theres so many changes when I changed just one thing in the styl file. :P

### CODE REVIEW

File: templates/applications.html [hunk 1]
```diff
@@ -40,6 +40,7 @@ <h3 class="page-header">Applications for {{ page.title }}</h3>
               </select>
               <input type="submit" value="Apply" />
           </p>
+          <a href="download" class="btn download">Download to csv</a>
```
Comment (helenst) [inline templates/applications.html]: It would be better to use the `{% url  %}` tag here for the href :strawberry:

Comment (jinpark) [inline templates/applications.html]: Ahh, good call. Let me make that change

Comment (jinpark) [inline templates/applications.html]: Made the change :)

**Comment by helenst** [comment]
Looks awesome to me. I'll leave for somebody else to merge as I'm not actually sure about the CSS thing. :sparkling_heart:

**Comment by olasitarska** [comment]
Yeap, that is correct -- we commit css too, because we don't have a preproccessor as part of deploy yet.

**Comment by olasitarska** [comment]
Thanks @jinpark and @helenst for review! :sparkles:

**Pull request merged**